### ISSUE TITLE
Intake | The 'enter or select form box' and 'Which form' label is not read by assistive technology (End: 6/12)

### ISSUE THREAD

**Issue by msj0nes** [department-of-veterans-affairs/caseflow#14373]
# Accessibility Issue Description

#### Defects Found: 
_Defect 2:_ The name, role, state or value of user interface elements in the product cannot be understood.
_Defect 3:_ Labels and instructions are not provided when content requires user input.

#### Details 
- The label and instructions _'Which Form Are You Processing_' is not read by assistive technology programs. 
- The role for the _Enter or Select Form combo box_ is not read by assistive technology programs. Only the text enter or Select form is read. 

![image](https://user-images.githubusercontent.com/41164406/82693626-dc33ce00-9c2f-11ea-8f35-3bf244f68e92.png)

## Acceptance criteria
- [ ] Testing should conclude the **enter or select form box** is read using assistive technology. 
- [ ] Testing should conclude the **Which Form Are You Processing** is read using assistive technology.

**Comment by Sjones352** [comment]
@leikkisa and @jcq. Upon reviewing this ticket, this SelectForm is inheriting/using from the SearchableDropdown props. The SearchableDropdown component uses the react-select api https://react-select.com/props which sets the Selectform at a combobox role. Chrome + Voiceover does not  announce the "focused" option when using the arrow keys on the dropdown listbox. 

FireFox + Voiceover announces the "focused" option by announcing the `Which Form Are You Processing` and  `Enter or Select Form combo box` as text elements. We only see this combobox when the ramp-intake feature toggle is enabled. We need to revisit this and figure out if we are still processing ramp claims. 

![Screen Shot 2020-06-07 at 11.49.16 PM.png](https://images.zenhubusercontent.com/5b8fddd1b2cc7b6c78ece9f9/88bff0c5-41a7-4237-8e81-dbbf5ff0f793)

![Screen Shot 2020-06-07 at 11.48.41 PM.png](https://images.zenhubusercontent.com/5b8fddd1b2cc7b6c78ece9f9/4919dabf-182d-417d-be59-21170c02ef8d).

We need to look into refactoring this component to not use react-select because it is not fully ARIA compliant and because react properties and attributes are camelCased, aria-* should be hyphen-cased, see link https://reactjs.org/docs/accessibility.html.
**Code Snippet**
![Screen Shot 2020-06-07 at 11.13.33 PM.png](https://images.zenhubusercontent.com/5b8fddd1b2cc7b6c78ece9f9/244deadf-742d-464e-8043-e0547852fe9c)

The WAVE chrome tool recognizes them at aria-* however VoiceOver does not read them. This might  probably apply to different versions of chrome.
![Screen Shot 2020-06-07 at 10.56.26 PM.png](https://images.zenhubusercontent.com/5b8fddd1b2cc7b6c78ece9f9/03ac2392-7f14-4c0d-972a-8799177dad3e)

**Comment by jcq** [comment]
This might be a good time/reason to finally upgrade `react-select`; we're using a quite old version (1.0.0-rc.4) and I know they've addressed ARIA stuff on a number of releases since then. Can't guarantee it remedies this, but certainly couldn't hurt!

**Comment by Sjones352** [comment]
@jcq and @leikkisa I did some digging and both labels can be read by VoiceOver. I also used the WAVE chrome tool to find out what roles are being read. See attachment below. I think it is safe to say that the `react-select` upgrade fixed this ticket.

<img width="1644" alt="Screen Shot 2020-07-14 at 11 40 01 AM" src="https://user-images.githubusercontent.com/23080951/87446210-fe5e2280-c5c6-11ea-9b62-a06330525d4f.png">

**Comment by leikkisa** [comment]
Sweet thanks - moving to review.
